superset-frontend/packages/superset-ui-core/src/theme/Theme.tsx

```diff
@@ -22,12 +22,20 @@ import React from 'react';
 import { theme as antdThemeImport, ConfigProvider } from 'antd';
 import tinycolor from 'tinycolor2';
 
+// @fontsource/* v5.1+ doesn't play nice with eslint-import plugin v2.31+
+/* eslint-disable import/extensions */
```

Comment:
Not related to this PR directly, but now I'm wondering if we use `200` anywhere in the codebase.

Sees we use only these now -> https://github.com/apache/superset/blob/master/superset-frontend/packages/superset-ui-core/src/theme/Theme.tsx#L90-L92

Found a few hard-coded values that don't match what we import here too. Might be a good thing to clean this up.